superset-frontend/src/views/CRUD/data/dataset/AddDataset/EditDataset/UsageTab/UsageTab.test.tsx

```diff
@@ -0,0 +1,405 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import fetchMock from 'fetch-mock';
+import userEvent from '@testing-library/user-event';
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
+import { ChartListChart, getMockChart } from 'spec/fixtures/mockCharts';
+import ToastContainer from 'src/components/MessageToasts/ToastContainer';
+import DatasetUsage from '.';
+
+const DEFAULT_DATASET_ID = '1';
+const DEFAULT_ORDER_COLUMN = 'last_saved_at';
+const DEFAULT_ORDER_DIRECTION = 'desc';
+const DEFAULT_PAGE = 0;
+const DEFAULT_PAGE_SIZE = 25;
+
+const getChartResponse = (result: ChartListChart[]) => ({
+  count: result.length,
+  result,
+});
+
+const CHARTS_ENDPOINT = 'glob:*/api/v1/chart/?*';
+const mockChartsFetch = (response: any) => {
```

Comment:
```suggestion
const mockChartsFetch = (
  response:
    | Promise<unknown>
    | { count: number; result: ChartListChart[] }
    | number,
) => {
```
This `any` can be described with this definition.

Reply:
Thanks for catching!  Looks like I can also use `fetchMock.MockResponse` – do you think that's a better options?

Reply:
Oh yeah that's much cleaner 😁